import-export-cli/utils/fileIOUtils.go

```diff
@@ -331,6 +331,54 @@ func CopyDirectoryContents (src string, dst string) (err error)  {
 	return
 }
 
+// MoveDirectoryContentsToNewDirectory recursively moves all the content of a directory to a given directory
+// attempting to preserve permissions.
+// Source directory must exist,and the destination directory exist.
+// @param src source directory path
+// @param dst destiny directory path
+// @return error
```

Comment:
Have we used standard Go Lang documentation comments style? check:e.g.https://www.cloudhadoop.com/2018/12/golang-tutorials-comments-in-go-language.html
And also here why can't we use multi-line comment rather than single line comments?
This should be fixed in many other places in your PRs if applicable.

Reply:
Fixed with f75a26712774c479a96a139dc5fdc95d089bb104